app/containers/MessageComposer/components/SendThreadToChannel.tsx

```
@@ -55,6 +55,7 @@ export const SendThreadToChannel = (): React.ReactElement | null => {
 		if (alsoSendThreadToChannelUserPref === 'default') {
 			const db = database.active;
 			const observable = db.get('threads').query(Q.where('id', tmid)).observe();
+            // @ts-ignore: fix me
```

ESLint

[error] Replace `············` with `↹↹↹`

(prettier/prettier)